### PR TITLE
fix(common): add `serverTarget` for prerender

### LINKED ISSUE #2120: prerender fails on new angular-cli 12 app
using `@angular/cli@12.0.1`

`ng new app`
`ng add @nguniversal/express-engine`
`npm run build:ssr`
`npm run prerender`

gives the following error:
>>
```bash
Schema validation failed with the following errors:
  Data path "" must have required property 'serverTarget'
```
manually adding `"serverTarget": "appname:server:production"` to `angular.json` seems to work, but this  probably should be done by the `@nguniversal/express-engine` schematics

I get that there may be a configuration step I'm missing, but the documentation is very unclear about this, and I couldn't find additional info about the prerender npm task.

Thank you

### PR DESCRIPTION
**Pull request by bampakoa**
Add `serverTarget` for the prerender architect in the add schematic

Closes #2120

### CODE REVIEW

**Review by alan-agius4 — APPROVED** [review]
LGTM, thanks for your contribution 

Can you please format the code?

```
  yarn ng-dev format changed 
```

**Review by alan-agius4 — CHANGES_REQUESTED** [review]
Actually, this is not the correct schematic, the problematic one is https://github.com/angular/universal/blob/master/modules/express-engine/schematics/install/index.ts

The changes here should be reverted

File: modules/express-engine/schematics/install/index.ts [hunk 1]
```diff
@@ -118,9 +118,11 @@ function updateWorkspaceConfigRule(options: AddUniversalOptions): Rule {
         configurations: {
           production: {
             browserTarget: `${options.project}:build:production`,
+            serverTarget: `${projectName}:server:production`,
```
Comment (alan-agius4) [inline modules/express-engine/schematics/install/index.ts:121]:
```suggestion
            serverTarget: `${options.project}:server:production`,
```

File: modules/express-engine/schematics/install/index.ts [hunk 2]
```diff
@@ -118,9 +118,11 @@ function updateWorkspaceConfigRule(options: AddUniversalOptions): Rule {
...
           },
           development: {
             browserTarget: `${options.project}:build:development`,
+            serverTarget: `${projectName}:server:development`,
```
Comment (alan-agius4) [inline modules/express-engine/schematics/install/index.ts:125]:
```suggestion
            serverTarget: `${options.project}:server:development`,
```

File: modules/express-engine/schematics/install/index.ts [hunk 3]
```diff
@@ -118,9 +118,11 @@ function updateWorkspaceConfigRule(options: AddUniversalOptions): Rule {
         configurations: {
           production: {
             browserTarget: `${options.project}:build:production`,
+            serverTarget: `${projectName}:server:production`,
           },
           development: {
             browserTarget: `${options.project}:build:development`,
```
Comment (alan-agius4) [inline modules/express-engine/schematics/install/index.ts:124]: Mind changing this to projectName? 😀

**Review by alan-agius4 — APPROVED** [review]
LGTM

**Comment by bampakoa** [comment]
@alan-agius4 you mean to change `projectName` to `options.project` right?

**Comment by alan-agius4** [comment]
Other way.

**Comment by bampakoa** [comment]
My browser was idle and I did not notice that the PR was actually merged 😁 Should I make a separate PR for the change that you propose?

**Comment by alan-agius4** [comment]
Sure, feel free to do a refactor PR

**Pull request merged**